shard.yml

```diff
@@ -1,29 +1,29 @@
 name: lucky_flow
-version: 0.9.1
+version: 0.9.2
 
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
 
-crystal: ">= 1.4.0"
+crystal: "~> 1.4"
 
 license: MIT
 
 dependencies:
   selenium:
     github: crystal-loot/selenium.cr
-    version: ~> 0.12
+    version: ">= 0.12.1, < 0.13"
   webdrivers:
     github: crystal-loot/webdrivers.cr
-    version: ~> 0.4
+    version: ">= 0.4.3, < 0.5"
   habitat:
     github: luckyframework/habitat
-    version: ~> 0.4.7
+    version: ">= 0.4.7, < 0.5"
```

Comment:
I'm fine with this one, because we were already requiring it to be `0.4.7`, whereas the other ones were not.